posthog-core/src/index.ts

```diff
@@ -439,21 +436,17 @@ export abstract class PostHogCoreStateless {
    *** FEATURE FLAGS
    ***/
 
-  protected async getDecide(
+  protected async getFlags(
     distinctId: string,
     groups: Record<string, string | number> = {},
     personProperties: Record<string, string> = {},
     groupProperties: Record<string, Record<string, string>> = {},
-    extraPayload: Record<string, any> = {}
-  ): Promise<PostHogDecideResponse | undefined> {
+    extraPayload: Record<string, any> = {},
+    withConfig: boolean = false
+  ): Promise<PostHogFlagsResponse | undefined> {
     await this._initPromise
 
-    // Check if the API token is in the new flags rollout
-    // This is a temporary measure to ensure that we can still use the old flags API
-    // while we migrate to the new flags API
-    const useFlags = isTokenInRollout(this.apiKey, NEW_FLAGS_ROLLOUT_PERCENTAGE, NEW_FLAGS_EXCLUDED_HASHES)
-
-    const url = useFlags ? `${this.host}/flags/?v=2` : `${this.host}/decide/?v=4`
+    const url = `${this.host}/flags/?v=2${withConfig ? '&config=true' : ''}`
```

Comment:
@marandaneto here's how I'm handling the remote config stuff.  Since react native is the library that needs session recording + survey flags, I've added this `withConfig` param that gets passed into `getFlags`, and then I pass that value into the react native `reloadFeatureFlags` method like this: https://github.com/PostHog/posthog-js-lite/pull/523/files#diff-913c519c5a9403e850fc7361df3421a61903eeca00e294653890068376e15f4cR152-R154